src/stactools/ecmwf_forecast/kerchunk_helper_functions.py

```diff
@@ -0,0 +1,60 @@
+from kerchunk.combine import MultiZarrToZarr
+from kerchunk.grib2 import scan_grib
+from numcodecs.registry import register_codec
+from stactools.ecmwf_forecast.range_codec import Range
+
+import base64
+import fsspec
+
+register_codec(Range)
```

Comment:
We'll want to update `pyproject.toml`. to register this.

Reply:
Updated setup.cfg to hopefully register it.